Formula/postgresql.rb

```diff
@@ -72,6 +79,12 @@ def install
     # which does not work on CLT-only installs.
     args << "PG_SYSROOT=#{MacOS.sdk_path}" if MacOS.sdk_root_needed?
 
+    on_linux do
+      # rebuild `configure` after patching
+      # (remove if patch block not needed)
+      system "autoreconf", "-ivf"
```

Comment:
Don't we need an `autoconf` dependency for this?

Reply:
Whoops, thanks for the catch! Fixing...